pkg/scaffold/v1/manager/apis.go

```diff
@@ -21,15 +21,16 @@ import (
 	"path/filepath"
 	"strings"
 
-	"sigs.k8s.io/kubebuilder/pkg/scaffold/input"
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
 )
 
-var _ input.File = &APIs{}
+var _ file.Template = &APIs{}
 
 // APIs scaffolds a apis.go to register types with a Scheme
 type APIs struct {
-	input.Input
-
+	file.Input
+	// BoilerplatePath is the path to the boilerplate file
+	BoilerplatePath string
```

Comment:
Is this used anywhere?

Reply:
A couple lines below to compute a relative path.